corehq/apps/app_manager/templates/app_manager/v2/module_view.html

```diff
@@ -292,6 +295,11 @@
               </div>
             {% endif %}
           {% endfor %}
+          {% if details %}
+            <div class="tab-pane" id="module-actions">
+                {% include 'app_manager/v1/partials/module_actions.html' %}
```

Comment:
This might as well include the v2 version of module_actions.html instead of v1. Not that it matters now since they're identical, but will be confusing if that changes.